svf/include/Graphs/PTACallGraph.h

```diff
@@ -263,7 +264,7 @@ class PTACallGraph : public GenericPTACallGraphTy
     static CallSiteID totalCallSiteNum;	///< CallSiteIDs, start from 1;
 
 protected:
-    FunToCallGraphNodeMap funToCallGraphNodeMap; ///< Call Graph node map
+    CgNodeToPTACallGraphNodeMap cgNodeToPtaCallGraphNodeMap; ///< Call Graph node map
```

Comment:
We don't need this map. We can use id to get ptacallgraphnode from callgraphnode